modules/services/servers/src/main/java/au/com/shiftyjelly/pocketcasts/servers/sync/SyncServerManager.kt

```diff
@@ -154,19 +160,8 @@ open class SyncServerManager @Inject constructor(
         server.getLastSyncAt(addBearer(token), buildBasicRequest())
             .map { response -> response.lastSyncAt ?: "" }
 
-    fun getHomeFolder(token: AccessToken): Single<PodcastListResponse> =
-        server.getPodcastList(addBearer(token), buildBasicRequest()).map { response ->
-            response.copy(podcasts = removeHomeFolderUuid(response.podcasts), folders = response.folders)
-        }
-
-    private fun removeHomeFolderUuid(podcasts: List<PodcastResponse>?): List<PodcastResponse>? =
-        podcasts?.map { podcast ->
-            if (podcast.folderUuid != null && podcast.folderUuid == Folder.homeFolderUuid) {
-                podcast.copy(folderUuid = null)
```

Comment:
Because Google's protoc implementation doesn't allow to set `null` value here I had to move it to `PodcastSyncProcess` when applying properties to podcasts.